src/ripple/rpc/impl/ServerHandlerImp.cpp

```diff
@@ -578,25 +578,33 @@ ServerHandlerImp::processRequest (Port const& port,
     bool batch = false;
     if (jsonOrig.isMember(jss::method) && jsonOrig[jss::method] == "batch")
         batch = true;
-    auto size = batch ? jsonOrig[jss::params].size() : 1;
+    auto size = batch
+              ? jsonOrig.isMember(jss::params)
+                ? jsonOrig[jss::params].size() : 0
```

Comment:
could `jss::params` be an object instead of an array (malicious input)? If yes, then I think the index access below will assert in that case.

Reply:
I'll verify, but I believe  `jss::params` is added as part of converting a batch JSON call, so I think it might not crash currently. Regardless, I'll add a test to confirm and add a check to ensure that requirement is true.